### PR TITLE
prevent users from running into dalli bug

### PR DESCRIPTION
**Pull request by grosser**
@pschambacher @ggrossman 

see https://github.com/petergoldstein/dalli/pull/481

### Risks
 - None

### CODE REVIEW

File: lib/prop/limiter.rb [hunk 1]
```diff
@@ -23,6 +23,14 @@ def cache=(cache)
           next if cache.respond_to?(method)
           raise ArgumentError, "Cache needs to respond to #{method}"
         end
+
+        # https://github.com/petergoldstein/dalli/pull/481
+        if defined?(ActiveSupport::Cache::DalliStore) &&
+            cache.is_a?(ActiveSupport::Cache::DalliStore) &&
+            Dalli::VERSION <= "2.7.4"
```
Comment (jish) [inline lib/prop/limiter.rb]:
```ruby
"2.10" < "2.7.4"
# => true
```

Comment (grosser) [inline lib/prop/limiter.rb]: yeah ok ...

Comment (grosser) [inline lib/prop/limiter.rb]: fixed

**Comment by pschambacher** [comment]
:+1: LGTM

**Pull request merged**